### DISCUSSION TITLE
Running Swanctl with mode=tunnel

### DISCUSSION THREAD

**Discussion by musaleh17** [strongswan/strongswan#1555] [General]
Hi,

when enabling IPSEC mode to be a tunnel, do we can specify the encap header attributes, such as destination IP? 

Where I can find an example of conf file with tunnel mode? all the examples in the project include transport mode!

**Comment by tobiasbrunner** [comment]
> when enabling IPSEC mode to be a tunnel, do we can specify the encap header attributes, such as destination IP?

The outer IP addresses are the same as those for the IKE packets (i.e. the endpoints of the tunnel).

> Where I can find an example of conf file with tunnel mode? all the examples in the project include transport mode!

That's definitely not the case. Only very specific test scenarios and examples actually use transport mode (`mode = tunnel` is the default after all).
